src-built-in/components/toolbar/src/app.jsx

```diff
@@ -21,10 +21,12 @@ const Toolbar = () => {
 				{/* Workspace Management Menu */}
 				{/* App Menu */}
 			</div>
-			<div className="finsemble-toolbar-section center">
+			<ToolbarSection className="center" minWidth={115}>
+				<div className="divider"></div>
```

Comment:
You should use a self-closing tag.
```suggestion
				<div className="divider"/>
```